frappe_telegram/client.py

```diff
@@ -63,6 +65,36 @@ def send_file(file, filename=None, message=None, user=None, telegram_user=None,
     if not from_bot:
         from_bot = frappe.db.get_default(DEFAULT_TELEGRAM_BOT_KEY)
 
+    if isinstance(file, File):
+
+        file_path = frappe.get_site_path(
+            (("" if "/private/" in file.file_url else "/public") + file.file_url).strip("/"))
+
+        if os.path.exists(file_path):
+            file = open(file_path, 'rb')
+        else:
+            frappe.throw(frappe._("File specified in the File doc does not exist."),
+                         exc=FileNotFoundError)
+    elif isinstance(file, str) and "/files/" in file:
+
+        # If file is string, check that the url is internal
+
+        files = frappe.get_all(
+            "File",
+            filters=[["file_url", "=", file]]
+        )
+
+        if files:
+            file_path = frappe.get_site_path(
```

Comment:
Hmm, so we are circumventing the File DocType, and checking directly for the file. Hmmm.
Feels a bit weird, but I don't see any problems with it.
A file can be deleted while the File doc still exists, so this might be better in fact.